jenkins/job-configs/jenkins-daily-maintenance.yaml

```diff
@@ -63,3 +63,29 @@
                 </hudson.plugins.parameterizedtrigger.TriggerBuilder>
     triggers:
         - timed: '@daily'
+
+- job-template:
+    name: 'google-cloud-resource-cleanup'
+    description: 'Clean up leaked cloud resources. Test owner: test-infra maintainers.'
+    properties:
+        - build-discarder:
+            days-to-keep: 30
+    scm:
+    - git:
+        url: https://github.com/kubernetes/test-infra
+        branches:
+        - master
+        browser: githubweb
+        browser-url: https://github.com/kubernetes/test-infra
+        skip-tag: true
+    triggers:
+        - timed: '@daily'
```

Comment:
agreed that we should fix leaks, though we also don't want to block the SQ due to leaks.

I've updated the script to work on hours instead of days, and made it run every 3 hours. we can back out to 24h or whatever when things are more under control.